docs/source/user_guide/get_started/examples/environ_calculation.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+
+from aiida import orm
+from aiida.engine import run, submit
+import numpy as np
+from ase import Atoms
+from ase import units
+"""
+Calculation that recreates the environ calculation of
+CO on Pt(111) with a unit charge
+
+More details about the solvation models and potential corrections
+can be found here:
+
+O. Andreussi, I. Dabo and N. Marzari, J. Chem. Phys. 136,064102 (2012).
+I. Dabo et al. Phys.Rev. B 77, 115139 (2008)
+
+This system is taken directly from Example04 in the
+examples folder from Environ
+"""
+
+
+def calculator():
+    """
+    This is the SCF calculator, which is identical
+    and unchanged from what one would do in the case
+    of a standard SCF calculation - nothing environ
+    related. Remember to add in the tot_charge here
+    """
+    param_dict = {
+        'CONTROL': {
+            'calculation': 'scf',
+            'tprnfor': True,
+        },
+        'SYSTEM': {
+            'ecutwfc': 35,
+            'ecutrho': 280,
+            'occupations': 'smearing',
+            'smearing': 'cold',
+            'degauss': 0.03,
+            'tot_charge': 1,
+        },
+        'ELECTRONS': {
+            'conv_thr': 5e-9,
+            'electron_maxstep': 200,
+            'mixing_beta': 0.2,
+            'mixing_ndim': 15,
+            'diagonalization': 'david',
+        },
+    }
+
+    return param_dict
+
+
+def environ_calculator():
+    """
+    Environ parameters
+    """
+    param_dict = {
+        'ENVIRON': {
+            'verbose': 0,
+            'environ_thr': 1.0,
+            'environ_type': 'input',
+            'env_static_permittivity': 1,
+            'env_surface_tension': 0.0,
+            'env_pressure': 0.0,
+            'env_electrostatic': True,
+        },
+        'BOUNDARY': {
+            'solvent_mode': 'full',
+        },
+        'ELECTROSTATIC': {
+            'pbc_correction': 'parabolic',
+            'pbc_dim': 2,
+            'pbc_axis': 3,
+            'tol': 5e-13,
+        }
+    }
+    return param_dict
+
+
+def runner(code, structure):
+
+    ## Base calculation for testing
+    PwBaseWorkChain = WorkflowFactory('quantumespresso.pw.base')
+    builder = PwBaseWorkChain.get_builder()
+    builder.pw.structure = structure
+
+    builder.metadata.label = 'Single point environ'
+    builder.metadata.description = 'Testing calculation with environ for Pt(111) + CO'
```

Comment:
This hardcodes the structure even though the structure is mutable through the argument. Not a huge deal since it is an example, but still a bit weird.

Reply:
Yup, also corrected.